statvar_imports/ndap/nss_health/README.md

```diff
@@ -0,0 +1,15 @@
+source
+: https://ndap.niti.gov.in/catalogue
+How to download
+go to the website 
+go to the nss health dataset 
+downloade the datasets
+type of place:
+For eg: State
+Place Resolution
+Place Resolution if any
+### How to run:
```

Comment:
Pls add a description for each of the 3 metadata files, what each contains and in the how to run section, please describe how to download files for each metadata and how to invole the statvar processfor for each (what pvmap and other config to use)